pkg/auth.go

```diff
@@ -321,7 +321,22 @@ func (auth *Auth) CreateAccessToken(request services.CreateAccessTokenRequest) (
 
 // CreateJwtBearerToken creates a JwtBearerToken from the given CreateJwtBearerTokenRequest
 func (auth *Auth) CreateJwtBearerToken(request services.CreateJwtBearerTokenRequest) (*services.JwtBearerTokenResult, error) {
-	return auth.AccessTokenHandler.CreateJwtBearerToken(&request)
+	// todo add checks?
+	custodian, err := core.ParsePartyID(request.Custodian)
+	if err != nil {
+		return nil, err
+	}
+	endpointType := "oauth" // todo
+
+	epoints, err := auth.Registry.EndpointsByOrganizationAndType(custodian, &endpointType)
+	if err != nil {
+		return nil, err
+	}
+	if len(epoints) == 0 {
+		return nil, oauth.ErrMissingEndpoint
+	}
+
+	return auth.AccessTokenHandler.CreateJwtBearerToken(&request, string(epoints[0].Identifier))
```

Comment:
just taking the first, is that right? If it's not something solveable, maybe log a big fat warning when it occurs?

Reply:
I believe this will have to become unique once services and endpoints enter the arena.